ConfigurationSystem/Agent/GOCDB2CSAgent.py

```diff
@@ -0,0 +1,245 @@
+# $HeadURL$
+"""
+Module provides GOCDB2CSAgent functionality.
+"""
```

Comment:
Good coding. 
It's a detail, but you can use self.log instead of gLogger directly in each Agent, and the logs will be prepended with the agent name.